README.md

```diff
@@ -3,73 +3,91 @@
 
-# get an item's value
-project.get("my int")
-
-# by default, strings are assumed to be Markdown:
-project.put("my string", "Hello world!")
+## 👨‍💻 More examples
 
-# `put` overwrites previous data
-project.put("my string", "Hello again!")
+💡 Note that after launching the dashboard, you can keep modifying the current items or store new ones from your python code, and the dashboard will automatically be refreshed.
 
-# list all the keys in a project
-print(project.list_item_keys())
+Storing a `pandas` dataframe:
+```python
+import numpy as np
+import pandas as pd
 
-# delete an item
-project.delete_item("my int")
+my_df = pd.DataFrame(np.random.randn(3, 3))
+project.put("my_df", my_df)
 ```
 
-Then, in the directory containing your project, run the following command in your shell to start the UI locally:
-```bash
-python -m skore launch project.skore
+Storing a `matplotlib` figure:
+```python
+import matplotlib.pyplot as plt
+
+x = [0, 1, 2, 3, 4, 5]
+fig, ax = plt.subplots(figsize=(5, 3), layout="constrained")
+ax.plot(x)
+project.put("my_figure", fig)
 ```
-This will automatically open a browser at the UI's location.
-In the `Elements` tab on the left, you can visualize the stored items.
-Create a new `View`, then you can then add items into this view.
 
-💡 Note that after launching the dashboard, you can keep modifying current items or store new ones, and the dashboard will automatically be refreshed.
+Storing a `scikit-learn` fitted pipeline:
+```python
+from sklearn.datasets import load_diabetes
+from sklearn.linear_model import Lasso
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
+diabetes = load_diabetes()
+X = diabetes.data[:150]
+y = diabetes.target[:150]
+my_pipeline = Pipeline(
+    [("standard_scaler", StandardScaler()), ("lasso", Lasso(alpha=2))]
+)
+my_pipeline.fit(X, y)
+project.put("my_fitted_pipeline", my_pipeline)
+```
 
-👨‍🏫 For a complete introductory example, see our [basic usage notebook](https://github.com/probabl-ai/skore/blob/main/examples/basic_usage.ipynb).
-It shows you how to store all types of items: python lists and dictionaries, `numpy` arrays, `scikit-learn` fitted models, `matplotlib`, `altair`, and `plotly` figures, etc.
-The resulting `skore` report has been exported to [this HTML file](https://sylvaincom.github.io/files/probabl/skore/basic_usage.html).
+👨‍🏫 For a complete introductory example, see our [basic usage notebook](https://github.com/probabl-ai/skore/blob/main/examples/01_basic_usage.ipynb).
+It shows you how to store all types of items: python lists and dictionaries, `numpy` arrays, `pandas` dataframes, `scikit-learn` fitted models, figures (`matplotlib`, `altair`, and `plotly`), etc.
+The resulting `skore` report has been exported to [this HTML file](https://sylvaincom.github.io/files/probabl/skore/01_basic_usage.html).
```

Comment:
It's a minor detail. Can we please change "Brought to you by:" to "Brought to you by" (remove colon) at line 110?

Reply:
Done